task_qc_viewer/README.md

```diff
@@ -25,12 +22,20 @@ Launch the Viewer by typing `ipython task_qc.py session_UUID` , example:
 ```
 ipython task_qc.py c9fec76e-7a20-4da4-93ad-04510a89473b
```

Comment:
This example does not work

Reply:
This one works: task_qc_viewer gaelle$ ipython task_qc.py fece187f-b47f-4870-a1d6-619afe942a7d

Reply:
2020-10-09 15:49:58.580 INFO     [task_metrics.py:72] Session /Users/gaelle/Downloads/FlatIron/zadorlab/Subjects/CSH_ZAD_029/2020-09-17/001: **Running QC on behavior data...**
This step takes ages. (minutes)